File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.io.*;
import java.util.StringTokenizer;

public class Main {
    static int solution(int a, int b, int c) {
        int answer = 0;
        if (a == b && a == c) {
            answer = 10000 + a * 1000;
        } else if (a == b && a != c) {
            answer = 1000 + a * 100;
        } else if (a == c && a != b) {
            answer = 1000 + a * 100;
        } else if (b == c && b != a) {
            answer = 1000 + b * 100;
        } else if (a != b && a != c) {
            answer = Math.max(Math.max(a, b), c) * 100;
        }
        return answer;
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        StringTokenizer st = new StringTokenizer(br.readLine());
        int a = Integer.parseInt(st.nextToken());
        int b = Integer.parseInt(st.nextToken());
        int c = Integer.parseInt(st.nextToken());
        bw.write(String.valueOf(solution(a, b, c)));
        
        bw.flush();
        bw.close();
        br.close();
    }
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
import sys

a, b, c = map(int, sys.stdin.readline().split())

if a == b and a == c and b == c:
    sys.stdout.write(str(10000+a*1000))
elif (a == b and a != c) or (a == c and a != b):
    sys.stdout.write(str(1000+a*100))
elif b == c and b != a:
    sys.stdout.write(str(1000+b*100))
elif a != b and a != c and b != c:
    sys.stdout.write(str(max(max(a, b), c)*100))
